pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -359,7 +359,15 @@ func (s *ServiceImpl) addPluginToSection(c *contextmodel.ReqContext, treeRoot *n
 
 func (s *ServiceImpl) hasAccessToInclude(c *contextmodel.ReqContext, pluginID string) func(include *plugins.Includes) bool {
```

Comment:
this function is meant to check for permissions, I don't think it's the right place to look for the feature flag.